Game/Config/DefaultGame.ini

```diff
@@ -5,6 +5,7 @@ bSpatialNetworking=True
 
 [/Script/UnrealEd.ProjectPackagingSettings]
 +DirectoriesToAlwaysCook=(Path="Spatial")
++DirectoriesToAlwaysCook=(Path="/SpatialGDK/SpatialDebugger/")
```

Comment:
As discussed offline - think this should also be done as an engine change - can be complete as a follow up